integration-test/src/main/java/com/sequenceiq/it/cloudbreak/ClusterCreationTest.java

```diff
@@ -64,6 +66,12 @@ public void testClusterCreation(@Optional("it-cluster") String clusterName, @Opt
         if (Boolean.TRUE.equals(withRDSConfig)) {
             clusterRequest = setRDSConfiguration(itContext, clusterRequest);
         }
+
+        if (Boolean.TRUE.equals(withFs)) {
```

Comment:
The expression is equaliement with if (withFs), not need to wrap (same for withRDSConfig)